tools/prowgen/pkg/testdata/simple.yaml

```diff
@@ -44,9 +44,20 @@ jobs:
     resources: custom
     command: [prow/command.sh]
     repos: [istio/istio]
-    modifier: [presubmit_skipped]
```

Comment:
Not sure if there was a reason to remove this line, but noting it was removed in the test-data as well.

Reply:
it no longer is a real field - I made the yaml parser strict